pkg/pillar/cmd/upgradeconverter/applydefaultconfigitem.go

```diff
@@ -39,6 +47,11 @@ func applyDefaultConfigItem(ctxPtr *ucContext) error {
 		log.Noticef("No existing ConfigItemValueMap; creating %s with defaults",
 			newConfigItemFile)
 	}
+	keyData := readAuthorizedKeys(baseAuthorizedKeysFile)
```

Comment:
I still think this authorized_keys handling should be done in the new import function.

Reply:
Would it make sense to have this also return an exists boolean, so the os.Remove is determined by that boolean?

Reply:
done